src/main/java/dan200/computercraft/core/lua/LuaJLuaMachine.java

```diff
@@ -412,6 +415,7 @@ public void execute()
                                             }
                                             catch( Throwable t )
                                             {
+                                                ComputerCraft.log.error( "Error running task", t);
```

Comment:
Debug API which is only of use to developers should be passed with .trace() and silenced by default, no?